templates/taxi/manufacturer_list.html

```diff
@@ -1,33 +1,40 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <h1>Manufacturer List
-    </h1>
+  <h1>Manufacturer List<a style="float: right" href="{% url 'taxi:manufacturer-create' %}">+</a></h1>
 
-    {% if manufacturer_list %}
-      <table class="table">
+  {% if manufacturer_list %}
+    <table class="table">
+      <tr>
+        <th>ID</th>
+        <th>Name</th>
+        <th>Country</th>
+        <th>Update</th>
+        <th>Delete</th>
+      </tr>
+
+      {% for manufacturer in manufacturer_list %}
         <tr>
-          <th>ID</th>
-          <th>Name</th>
-          <th>Country</th>
+          <td>
+            {{ manufacturer.id }}
+          </td>
+          <td>
+            {{ manufacturer.name }}
+          </td>
+          <td>
+            {{ manufacturer.country }}
+          </td>
+          <td>
+            <a href="{% url 'taxi:manufacturer-update' pk=manufacturer.id %}">UPDATE</a>
+          </td>
+          <td>
+            <a href="{% url 'taxi:manufacturer-delete' pk=manufacturer.id %}">DELETE</a>
```

Comment:
Code Style issue: According to the checklist, it's recommended to add a `Cancel` button that will lead to the previous page the user was on.